Controller/Callback/Offsite.php

```diff
@@ -143,6 +144,9 @@ public function execute()
                     case Truemoney::CODE:
                         $dispPaymentMethod = "True Money";
                         break;
+                    case Truemoney::CODE:
```

Comment:
Truemoney?

Reply:
I did refactoring, just now... renamed offsite-offline... please wait..

Reply:
since it is `offline` this code is never reached